robot-core/src/main/java/org/obolibrary/robot/export/Row.java

```diff
@@ -275,6 +275,42 @@ public String toHTML(List<Column> columns, String split) {
     return sb.toString();
   }
 
+  /**
+   * @param columns
+   * @return
+   */
+  public String toHTMLList(List<Column> columns) {
+    StringBuilder sb = new StringBuilder();
+    sb.append("<ul>\n");
+    for (Column c : columns) {
+      String columnName = c.getDisplayName();
+      Cell cell = cells.getOrDefault(columnName, null);
+      if (cell != null) {
+        // Only add to HTML list if not null and values is not empty
+        List<String> values = cell.getDisplayValues();
+        if (values.size() == 0) {
+          continue;
+        }
+        sb.append("\t<li>\n");
```

Comment:
I'm not sure why we're using tabs for the HTML output, when we don't use tabs anywhere else in any of our projects (TSV doesn't count).

Reply:
I guess to make the HTML more legible. Should I take it out?

Reply:
I would like the code to use spaces for indentation, please. Two spaces for each level works well for HTML, which can get quite deep.